specification/sql/resource-manager/Microsoft.Sql/preview/2018-06-01-preview/LongTermRetentionManagedInstanceBackups.json

```diff
@@ -0,0 +1,920 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2018-06-01-preview",
+    "title": "SqlManagementClient",
+    "description": "The Azure SQL Database management API provides a RESTful set of web APIs that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases, servers, and other entities."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
```

Comment:
WHat about the PUT api for this? how are these created?

Reply:
These are created through the currently existing backup infrastructure.  Generally, full database backups are taken 1x/week and our workflow determines whether or not a full backup for a server/managed instance should be saved for long term retention.

Reply:
In other words, a user cannot "create" a new backup